apps/vehicles/config/config.exs

```diff
@@ -28,3 +28,7 @@ use Mix.Config
 # here (which is why it is important to import them last).
 #
 #     import_config "#{Mix.env}.exs"
+
+# let logs happen, but only show warning and up in console
+config :logger, level: :info
+config :logger, :console, level: :warn
```

Comment:
This is probably not what you want, since it'll affect all applications.

Reply:
I went ahead and removed this as it didn't help anyway. This one test pictured below often fails (seen in build 2 of this branch), despite clearly outputting the log it seeks. It seems sometimes capture_log works, and sometimes it doesn't...

<img width="745" alt="image" src="https://user-images.githubusercontent.com/2136286/111380531-db669f80-867a-11eb-870d-bea5a4aed3d4.png">